src/content/docs/synthetics/synthetic-monitoring/private-locations/containerized-private-minion-cpm-configuration.mdx

```diff
@@ -138,7 +138,7 @@ Now you can add `"require('async');"` into the [script](/docs/synthetics/new-rel
 
 ### Change `package.json` for custom modules [#change-package-json]
 
-Along with npm modules, you can also use [Node.js modules](/docs/synthetics/new-relic-synthetics/scripting-monitors/import-nodejs-modules). To change the custom modules used by your CPM, modify `package.json` and reboot the CPM. It will detect the change in configuration during the reboot, and then clean up and re-install.
+Along with hosted modules, you can also use [Node.js modules](/docs/synthetics/new-relic-synthetics/scripting-monitors/import-nodejs-modules). To change the custom modules used by your CPM, modify `package.json` and reboot the CPM. It will detect the change in configuration during the reboot, and then clean up and re-install.
```

Comment:
@keegoid-nr previously we refer to "custom node modules." Are "hosted modules" the same thing?

Reply:
@rhetoric101 "custom node modules" includes hosted and custom modules.
I think this would be more clear by saying `Along with custom and hosted modules, you can also use...`

Reply:
OK, I'm updating the text to this: "Along with custom and hosted modules...."